github/github.go

```diff
@@ -98,7 +234,7 @@ func (g *github) generateJWTToken() (string, error) {
 	claims := jwt.StandardClaims{
 		IssuedAt:  time.Now().Unix(),
 		ExpiresAt: time.Now().Add(time.Minute * 10).Unix(),
-		Issuer:    g.appID,
+		Issuer:    g.appId,
```

Comment:
comment: for this one I know it may be weird, but we were following the guides from Golang 
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
that states initialisms should be always uppercase. (and for FE I use `Id` instead, but all of the go repros are using `ID`)

if you want to change that we can have a discussion on that

Reply:
Great catch! I found that we were using `Id` elsewhere that is why I changed it

Reply:
I thought we were using `ID` lol
yeah but anyways we just pick 1 and stick with that, I am fine with both lol

Reply:
I think we should follow the community, so lets stick with `ID`